src/webaudio.js

```diff
@@ -298,17 +302,22 @@ export default class WebAudio extends util.Observer {
              * output. Here we create an HTMLAudioElement, connect the
              * webaudio stream to that element and setSinkId there.
              */
-            let audio = new window.Audio();
+            if (!this.audio) {
+                this.audio = new window.Audio();
+                this.audio.autoplay = true;
+            }
+            let audio = this.audio;
```

Comment:
Hi @kross-italk! Thanks for your commit

Quick suggestion: since we already have `this.audio` variable, I don't think we need to reassign it to another var, just use it directly in this method